bucket/meilisearch.json

```diff
@@ -0,0 +1,26 @@
+{
+    "homepage": "https://github.com/meilisearch/meilisearch",
+    "version": "1.7.3",
+    "description": "A lightning-fast search engine that fits effortlessly into your apps, websites, and workflow.",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/meilisearch/meilisearch/releases/download/v1.7.3/meilisearch-windows-amd64.exe",
+            "hash": "a9801a60ee54ce6b1c3d19948031472bacad1f2af4aaf007848af8f0474efa13",
+            "bin": [
+                [
+                    "meilisearch-windows-amd64.exe",
+                    "meilisearch"
+                ]
+            ]
```

Comment:
Usually we put the `bin` section outside of `architecture` because it's normally the same for all architectures. To simplify this we can put `#/meilisearch.exe` on the end of the `url` in `architecture` and `autoupdate` so that Scoop renames the file and then `bin` can just be `"meilisearch.exe"`